dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -376,9 +376,14 @@ def test_check_meta_typename():
     assert "pandas" in str(info.value)
 
 
-def test_is_dataframe_like():
+def test_is_dataframe_like(monkeypatch):
     df = pd.DataFrame({"x": [1, 2, 3]})
     ddf = dd.from_pandas(df, npartitions=1)
+
+    if not hasattr(pd.DataFrame, "value_counts"):
+        # TODO: Set the pandas version this shows up in (likely 1.0)
+        monkeypatch.setattr(pd.DataFrame, "value_counts", lambda x: None, raising=False)
```

Comment:
Do we want to keep this `monkeypatch` long term, or is it here only to demonstrate this test will pass once https://github.com/pandas-dev/pandas/pull/27350 is merged? The changes to `is_dataframe_like()` look to be backwards compatible

Reply:
Long term, we should remove it.

It occurs to me that we should test both with and without the patch. I can get to that next week unless someone beats me to it.